### PR TITLE
Feature/add global settings

### PR DESCRIPTION
**Pull request by ksobon**
### READY

This is a start to new global settings UI where we are going to allow users to set their MC global preferences. Things to set from here: 

- path to website link
- more coming....

This was just a start to get the boiler plate stuff out of the way. We can merge it and then I got another one coming that will take care of the next thing which was office locations.

### CODE REVIEW

**Comment by ksobon** [comment]
@djsiroky how do you want to go about the Settings page? There are going to be a handful of new settings that we need to add to this page that will affect the app all over the place. I can either submit the PRs for review in piece-meal fashion so that they are smaller and easier to test/validate, or we can wait for one gigantic PR when all is said and done. Preference?

**Comment by djsiroky** [comment]
@ksobon Single-function/piecemeal like this is good with me. It's helpful to have all files be related to a specific function/area.

**Comment by ksobon** [comment]
@djsiroky ok, so this one is ready to go. Have a look at it, and approve it if you don't see anything wrong or missing, and I will get it merged.

File: app/controller/settings.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    },
+
+    /**
+     * Updates given Settings by Id.
+     * @param req
+     * @param res
+     */
+    update: function(req, res) {
+        var id = req.params.id;
+        Settings.update({ '_id': id }, req.body, { upsert: true }, function (err, response){
```
Comment (djsiroky) [inline app/controller/settings.js]: I'm getting a `DeprecationWarning: collection.update is deprecated. Use updateOne, updateMany, or bulkWrite instead.` on this. Looks like `updateOne` should work for this.

File: public/angular-app/settings/settings.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                                   class="form-control"
+                                   id="httpAddress"
+                                   placeholder="www.missioncontrol.com"
+                                   ng-model="vm.settings.httpAddress" required>
+                               <div ng-show="form.$submitted || form.link.$touched">
+                                   <div ng-show="form.link.$error.required" class="text-danger">(*) Website Address is required.</div>
+                               </div>
+                           </div>
+                       </fieldset>
+                       <!-- <fieldset class="form-group">
```
Comment (djsiroky) [inline public/angular-app/settings/settings.html]: Can this whole commented out section be removed? It looks like it's from the project creation form.

Comment (ksobon) [reply public/angular-app/settings/settings.html]: I think I removed that in the next PR.

File: app/controller/settings.js (hunk 1 above)
Comment (ksobon) [reply app/controller/settings.js]: Fixed.

**Pull request closed without merging**